### ISSUE TITLE
Expand Categories on search

### ISSUE THREAD

**Issue by Nicola1971** [pmfx/ElementsInTree#10]
downloaded latest version 1.2.1, but do not like how the search works.

In my opinion, when you search, collapsed categories should expand, like in my old demo:
https://www.youtube.com/watch?v=4RpJPa_CWAU

**Comment by Deesen** [comment]
But doesnt work with "remember collapsed"..

**Comment by Deesen** [comment]
I tried to make it work but failed and I thought its not that important. The tricky part is to close categories back to original collapsed states when leaving search-field focus. Also when displaying search results, should categories still be clickable/collapsible or not. Should the state be saved or not? Etc..

**Comment by Deesen** [comment]
But let me try again :-)

**Comment by Deesen** [comment]
What to say.. new day, and suddenly the right idea comes up.. do you want to test?

**Comment by Nicola1971** [comment]
Just Tested. Works perfect now :)

**Comment by Deesen** [comment]
When searchfield is focused, click on a closed category.. :-D

**Comment by pmfx** [comment]
@Deesen is it finished? I wonder if I should copy from you or you will create PR.
